modules/payloads/stages/windows/custom.rb

```diff
@@ -0,0 +1,38 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+module MetasploitModule
+  include Msf::Payload::Windows
+  include Msf::Payload::Custom
+  include Msf::Payload::Custom::Options
+
+  def initialize(info = {})
+    super(
+      merge_info(
+        info,
+        'Name' => 'Windows shellcode stager',
+        'Description' => 'Custom shellcode stager',
```

Comment:
This is a stage not a stager right?
```suggestion
        'Name' => 'Windows shellcode stage',
        'Description' => 'Custom shellcode stage',
```

Reply:
Hmmmmm...... that's fair.  It is stored in the stages folder, but when you call generate, you get a stager for shellcode.